test/mp/test/foo.h

```diff
@@ -21,7 +21,7 @@ struct FooStruct
     std::vector<bool> vbool;
 };
 
-enum class FooEnum : int { ONE = 1, TWO = 2, };
+enum class FooEnum : int { ONE = 1, TWO = 2, }; // NOLINT(performance-enum-size)
```

Comment:
In commit "clang-tidy: Suppress `performance-enum-size` check warning" (19d7537cb52eef623439e1dd6b562524021ae1a4)

This seems like another thing that would be nice to fix instead of nolint. These things can be done as followups since it is easy to search for nolint.

Reply:
This works:

```diff
-enum class FooEnum : int { ONE = 1, TWO = 2, }; // NOLINT(performance-enum-size)
+enum class FooEnum : uint8_t { ONE = 1, TWO = 2, };
```

Reply:
re: https://github.com/chaincodelabs/libmultiprocess/pull/144#discussion_r1941401788

This is done in #152